tests/test_pipeline.py

```diff
@@ -133,9 +133,9 @@ def test_predict_relations_with_novelties(self):
     def test_predict_heads_with_novelties(self):
         """Test scoring heads with labeling as novel w.r.t. training and testing."""
         heads_df = get_head_prediction_df(
-            self.model,
-            "conferences",
-            "brazil",
+            model=self.model,
```

Comment:
@mberr this is broken now...

Reply:
fixed in https://github.com/pykeen/pykeen/pull/949/commits/8fe5355a523ed73cd8559490746d9aac09ca6783

Reply:
thank you!!